### PR TITLE
feat(productcard): add itemLabel for productCard

### LINKED ISSUE #15: State Button
**Task title**
Create the state Button

**Task description**
Design a button with three distinct states: "For Sale," "For Borrow," and "For Borrow/Sale." Users can easily toggle between these states, each triggering a specific action: buying, borrowing, or a combination of both. Ensure clarity and user-friendliness in the button's design and functionality.

**Screenshots**
image

**Link to the component on Figma**
[Figma Link
](https://www.figma.com/file/FYZIsFvg1rLu50UnriUN8Y/Student-Store-Project?type=design&node-id=70-793&mode=design&t=0DdQfVRHQ85SzzLb-0)

### PR DESCRIPTION
**Pull request by Bolphunga**
**Changes made**:
Added item label for the type of product offer.
Defined utility classes for 4 extra fonts and fixed broken ones.
Modified layout and tailwind.config files to make fonts convenient to use with variables.

Component Image:
image
image
image

# Related Issue
Resolves #15 

# Related Issue

- Resolve #15

### CODE REVIEW

File: tailwind.config.js [hunk 1]
```diff
@@ -17,17 +17,24 @@ module.exports = {
...
+                poppins: ["var(--font-poppins)"],
             },
             letterSpacing: {
                 title: "tracking-normal",
                 navigation: "tracking-normal",
             },
         },
     },
-    plugins: [],
+    plugins: [require("@tailwindcss/aspect-ratio")],
```
Comment (0m3ga13) [inline tailwind.config.js:41]: did you use this?

Comment (Bolphunga) [reply tailwind.config.js:41]: Yes. Apparently, the fonts need it as well.

**Review by 0m3ga13 — APPROVED** [review]
so it's done ✅

**Review by medshk — APPROVED** [review]
PR permitted

**Pull request merged**